File: Codelearn/RACING TEAM CONTEST #4/countLike.cpp
int countLike(std::vector<int> arr)
{
   int dan3002 = -1;
    for (int funix=0;funix<arr.size();funix++) {
        if (arr[funix] > dan3002) dan3002 = arr[funix];
    }
    vector <int> times(dan3002+1,0);
    for (auto a : arr) {
        times[a]++;
    }
     
    int nguyendinhanh = 0;
    for (int kirby=0;kirby<times.size();kirby++) {
         if (times[kirby] % 2 != 0) nguyendinhanh++;
    }
    return nguyendinhanh;
}
 

File: Codelearn/Traning/countLike.js
function countLike(arr) {
   let map = new Map();

   for(let i of arr){
      map.set(i, (map.get(i) || 0) + 1);
   }

   let count = 0;
   map.forEach((item, i) => {
      count += item % 2 != 0 ? 1 : 0;
   });

   return count;

}